src/generate-qr-code-image.js

```diff
@@ -0,0 +1,33 @@
+import Syncano from 'syncano-server';
+import qr from 'qr-image';
+
+export default async (ctx) => {
+  const { response } = Syncano(ctx);
+
+  const checkQRImage = () => new Promise((resolve, reject) => {
+    if (!ctx.args.text) {
+      return reject(response.json({
```

Comment:
you are using the `response.json()` method in the catch clause below. You can reject with just a message that will be handled in the try catch.

Reply:
fixed